xml/operating_system.xml

```diff
@@ -693,6 +702,7 @@
   <fingerprint pattern="^(?i:(?:Juniper\s?)?(Junos|Junos OS|ScreenOS)\s?(\d+?(?:\.\d+?)*?)?)$">
     <description>Juniper</description>
     <example os.family="Junos" os.product="Junos">Junos</example>
+    <example os.family="Junos" os.product="Junos" os.version="1.0">Junos 1.0</example>
```

Comment:
[Junos OS Dates & Milestones](https://support.juniper.net/support/eol/software/junos/) appears to show a limit history given the initial release of Junos OS was in 1998, but I didn't find anything quickly showing releases in 1998. Perhaps we use a more current version number from the list?

Reply:
Fixed the regular expression and added a couple examples.